### ISSUE TITLE
Need accessible version of radio station table

### ISSUE THREAD

**Issue by mitchkeller** [oscarlevin/discrete-book#124]
At the 2022 Braille books workshop, we went hunting for figures and tables to try embossing. This table
https://github.com/oscarlevin/discrete-book/blob/61d2369f09dee6e473ab3cc44c9c0745a83bcd31/source/sec_gt-coloring.ptx#L351
is an image. I can see in the code why that is, but it is of course totally inaccessible as an image.

**Comment by oscarlevin** [comment]
Thanks.  I agree, and I'm not sure how to fix it.  I mean, I know how to make a table, but it doesn't fit on a page then.  Although I guess I should try it again now that more things are "squishy"

**Comment by mitchkeller** [comment]
Maybe the `tabular` should be the default (so that HTML is accessible) and you use a `version` to put the alternate format into the LaTeX? It feels like at the moment, the LaTeX is being allowed to drive the bus at the expense of your HTML's accessibility.
